.github/workflows/test-phpunit.yml

```diff
@@ -0,0 +1,95 @@
+name: Test Framework
+
+on:
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+
+  tests:
+    runs-on: ubuntu-latest
+
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+
+    name: PHP ${{ matrix.php-versions }} - ${{ matrix.db-platforms }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.2', '7.3', '7.4']
+        db-platforms: ['sqlite', 'mysqli', 'postgres']
+
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+          MYSQL_DATABASE: test
+        ports:
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: test
+        ports:
+          - 5432:5432
+        options: --health-cmd=pg_isready --health-interval=10s --health-timeout=5s --health-retries=3
+      redis:
+        image: redis
+        ports:
+          - 6379:6379
+        options: --health-cmd "redis-cli ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP, with composer and extensions
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+          tools: composer:v2
+          extensions: imagick
+          coverage: xdebug
+        env:
+          update: true
+
+      - name: Install Memcached
+        uses: niden/actions-memcached@v7
+
+      - name: Get composer cache directory
+        id: composercache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composercache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: ${{ runner.os }}-composer-
+
+      - name: Install dependencies
+        run: composer install --no-progress --no-suggest --no-interaction --prefer-dist --optimize-autoloader
+        env:
+          COMPOSER_AUTH: ${{ secrets.COMPOSER_AUTH }}     
+
+      - name: Test with PHPUnit
+        run: script -e -c "vendor/bin/phpunit -v"
+        env:
+          DB: ${{ matrix.db-platforms }}
+          TERM: xterm-256color
+
+      - name: Run Coveralls
+        if: ${{ success() }}
+        run: |
+          composer global require twinh/php-coveralls
```

Comment:
should can use `php-coveralls/php-coveralls` instead?

Reply:
I wanted to, but as I understand it's not yet fully supported https://github.com/php-coveralls/php-coveralls/issues/273

Reply:
I see there is now a pending PR for this issue, so of course, if they merge it we will switch to `php-coveralls`.

Reply:
@michalsn , the change was incorporated into upstream of php-coveralls , with big thanks for @twinh